.github/workflows/integration_test_legacy.yaml

```diff
@@ -0,0 +1,39 @@
+name: Integration Tests (Legacy)
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+      - 'next'
+  push:
+    branches:
+      - 'main'
+      - 'next'
+    tags:
+      - 'v*.*.*'
+
+jobs:
```

Comment:
Why not define legacy and non-legacy jobs in the same file?
Since the job definitions are mostly identical (with no way to de-duplicate that's immediately obvious to me), keeping them in one file might help with visibility that if one gets changed then the other needs to get changed as well.

Reply:
No reason not to, resolved by a2cac83b8f9bdc3dcbee2ba618bc8d464d0806e8 :+1: